config/default.js

```diff
@@ -25,6 +25,13 @@ module.exports = {
     clientSecret: "<clientSecret>",
   },
 
+  emailSubscriptionCredentials: {
+    user: "<RDS_EMAIL>",
```

Comment:
- should we name this email instead of user as it is not very indicative of what it represents?